src/app/shared/NoteActions.svelte

```diff
@@ -138,7 +133,7 @@
         id: note.id,
         anonymous: Boolean(note.wrap),
       })
-      .cx({callback: addToContext})
+      .cx({callback: repository.publish})
```

Comment:
This can actually be removed. We might be able to get rid of `callback` from Zap as well if it's not used anywhere else.